backend/jyll/settings.py

```diff
@@ -37,8 +37,11 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'users.apps.UsersConfig',
 ]
 
+AUTH_USER_MODEL = 'users.User'  # replacing User by CustomUser
```

Comment:
Le commentaire doit aussi être modifié, je propose `# replacing the default django User`